modules/contentbox/modules/contentbox-admin/modules/contentbox-filebrowser/handlers/Home.cfc

```diff
@@ -110,6 +110,23 @@ component{
 		// get directory listing.
 		prc.fbqListing = directoryList( prc.fbCurrentRoot, false, "query", prc.fbSettings.extensionFilter, "#prc.fbPreferences.sorting#" );
 
+		if(prc.fbPreferences.listFolder eq "dir" and structkeyexists(rc,"sorting")){
+			if(prc.fbPreferences.sorting eq "lastmodified"){
+				prc.fbPreferences.sorting ="datelastmodified"
+			}
+			var fileListQuery = new query();
```

Comment:
Can we update this to be a litte more streamline:

```
var fileListQuery = new Query( 
	dbType = "query",
	qry = prc.fbqListing,
	sql = "select * from qry where type=:dir order by #prc.fbPreferences.sorting#"
);

fileListQuery.addParam( name="dir", value="dir" );
```

Reply:
@lmajano I have updated the codes.can you Please review it...